samples/subsys/usb/hid-mouse/README.rst

```diff
@@ -0,0 +1,70 @@
+.. _usb_hid-mouse:
+
+USB HID mouse Sample Application
+####################################
+
+Overview
+********
+
+This sample app demonstrates use of a USB HID driver provided by the zephyr
+project, enumerate a board into a mouse. But that mouse only has left button
+and can't move.
+This sample can be found under :file:`samples/subsys/usb/hid-mouse` in the
+Zephyr project tree.
+
+Requirements
+************
+
+This project requires an USB device driver, and there must has at least one
+gpio button in your board.
+
+Running
+********************
+
+Plug the board into a host device, for example, a PC running Linux.
+The board will be detected as shown by the Linux dmesg command:
+
+.. code-block:: console
+
+    usb 2-2: new full-speed USB device number 2 using at91_ohci
+    usb 2-2: New USB device found, idVendor=2fe3, idProduct=0100
+    usb 2-2: New USB device strings: Mfr=1, Product=2, SerialNumber=3
+    usb 2-2: Product: USB-DEV
+    usb 2-2: Manufacturer: ZEPHYR
+    usb 2-2: SerialNumber: 0.01
+    input: ZEPHYR USB-DEV as /devices/soc0/ahb/600000.ohci/usb2/2-2/2-2:1.0/0003:2FE3:0100.0001/input/input0
+    hid-generic 0003:2FE3:0100.0001: input: USB HID v1.10 Mouse [ZEPHYR USB-DEV] on usb-at91-2/input0
+
+Monitor mouse events by using evtest tools:
+
+.. code-block:: console
+
+    # evtest /dev/input/event0
+    Input driver version is 1.0.1
+    Input device ID: bus 0x3 vendor 0x2fe3 product 0x100 version 0x110
+    Input device name: "ZEPHYR USB-DEV"
+    Supported events:
+      Event type 0 (EV_SYN)
+      Event type 1 (EV_KEY)
+        Event code 272 (BTN_LEFT)
+        Event code 273 (BTN_RIGHT)
+        Event code 274 (BTN_MIDDLE)
+      Event type 2 (EV_REL)
+        Event code 0 (REL_X)
+        Event code 1 (REL_Y)
+        Event code 8 (REL_WHEEL)
+      Event type 4 (EV_MSC)
+        Event code 4 (MSC_SCAN)
+    Properties:
+    Testing ... (interrupt to exit)
+
+Press the button in your board, will get those informations:
```

Comment:
Change to:

    When you press the button on your board, it will act as if the left
    mouse button was pressed, and this information will be displayed
    by ``evtest``: